javascript/javascript1/week2/week2-guzide/weather-wear/WeatherWear.js

```diff
@@ -0,0 +1,22 @@
+function chooseClothes(temperature) {
+  switch (true) {
```

Comment:
I think there is a mistake in this line :)
try to test the code for each case, this fuction does not work as intended even witn the values  you test for.

Reply:
I double checked each condition (5 conditions) and couldn't find any problem.
for these inputs: `chooseClothes(5);
chooseClothes(40);
chooseClothes(15);
chooseClothes(28);
chooseClothes(-40);` I get those ouptputs: `Wear 2 layers; outer layer can be a coat
You don't have to wear anything!
Wear 2 layers; outer layer can be a jacket
Wear 1 thin layer
Wear at least 3 layers and use at least 1 termal layer`